CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
```

Comment:
I think `changelog: add missing summary messages` is more correct

Reply:
Thank you, fixed!